### ISSUE TITLE
Cannot cancel alarm when there are two at the same time

### ISSUE THREAD

**Issue by nermolov** [MycroftAI/skill-alarm#20]
```
status of alarms
>> you have 2 active alarms, one for 1 p.m.. one for 1 p.m..
cancel my 1 p.m. alarm
>> you have {} alarms similar to that. which one are you referring too.
```

**Comment by penrods** [comment]
Should be resolved by #26 once it is merged into the Mycroft Skills repo

**Comment by penrods** [comment]
All merged!  I tried this out and it works properly now.
